Payload_Type/Apollo/agent_code/ScreenshotLoaderStub/Program.cs

```diff
@@ -0,0 +1,150 @@
+﻿//#define POWERPICK
+
+using System;
+using System.Management.Automation.Runspaces;
+using System.IO.Pipes;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Security.AccessControl;
+using IPC;
+using System.IO;
+using System.Drawing;
+using System;
+using System.Windows.Forms;
+using System.Collections.Generic;
+
+
+
+
+// Inject this assembly into the sacrificial process
+namespace ScreenshotRunner
+{
+    public class Program
+    {
+        static BinaryFormatter bf = new BinaryFormatter();
```

Comment:
Because no IPC message struct was created, we can't apply any security settings to this BinaryFormatter object and cannot send fine-grained messages.

Reply:
Fixed in 389f9c3